cpp/include/raft/mdarray.hpp

```diff
@@ -647,4 +687,30 @@ auto make_device_vector(raft::handle_t const& handle, size_t n)
 {
   return make_device_vector<ElementType>(n, handle.get_stream());
 }
+
+template <typename host_mdspan_type, typename = is_host_mdspan_t<host_mdspan_type>>
```

Comment:
Should the second template check with `is_host_mdspan_t` be a `static_assert` to display an error message to the user?